docs/benchmarks/dqc.md

```diff
@@ -0,0 +1,67 @@
+# Stats
+
+In this section, we benchmark between backends to solve a partial differential equation using a Differential Quantum Circuit ([DQC](https://arxiv.org/abs/2011.10395)). The underlying ansatz is the hardware-efficient-ansatz.
+The underlying gradient-based Adam optimizer is run for $20$ iterations.
+The example is taken from the `pyqtorch` and `horqrux` documentation.
+The circuits are defined over $4, 10$ qubits $R=5$ for avoiding long jobs time on Github.
+
+
+# DQC
+
+Here are the median execution times for VQE. We compare optimizing with `PyQTorch` against optimizing with `Horqrux` and jitting.
```

Comment:
I don't think it makes for a nice transition from DQC to VQE. This has to be explained a bit more.